nemo/collections/tts/modules/fastpitch.py

```diff
@@ -155,13 +215,15 @@ def __init__(
         self.pitch_predictor = pitch_predictor
         self.energy_predictor = energy_predictor
         self.aligner = aligner
+        self.speaker_encoder = speaker_encoder
         self.learn_alignment = aligner is not None
         self.use_duration_predictor = True
         self.binarize = False
         self.speaker_emb_condition_prosody = speaker_emb_condition_prosody
         self.speaker_emb_condition_decoder = speaker_emb_condition_decoder
         self.speaker_emb_condition_aligner = speaker_emb_condition_aligner
 
+        # TODO: combine self.speaker_emb with self.speaker_encoder
```

Comment:
What's the plan for this TODO? Any timeline in mind?

Reply:
Once we merge this PR, we can raise another PR for this -> this might take some time around 2 weeks to do this because even thought the code change is easy, we have to change the state_dict of the existing NGC checkpoints, test them and upload them.